catalog/ui/search-ui/standard/src/main/webapp/js/model/Query.js

```diff
@@ -490,12 +389,24 @@ define([
             },
 
             buildSearchData: function(){
-                var data = this.toJSON();
+                var data = _.clone(this.attributes);
```

Comment:
I would suggest still calling `this.toJSON()` just in case someone overrides it again in the future.